django/db/models/query.py

```diff
@@ -1701,6 +1702,21 @@ def using(self, alias):
         clone._db = alias
         return clone
 
+    def comment(self, message):
+        """Adds a comment to be inserted into the query."""
+        clone = self._clone()
+        # Sanitize comment by recursively discarding /* and */ delimiters.
+        num_subs = 1
+        while num_subs:
```

Comment:
Since the Python `re` module do not (to my knowledge) support recursive patters (`(?R)`) it felt easiest to implement the recursion in Python to keep it simple and reviewable, but if anyone has any tricks up their sleeve I'm more than welcome to get rid of that while-loop.

Reply:
Do we need to do this recursively? I think we may be more restrictive and raise an error when `message` contains `\*`, `*/` or `--`.